s3cache/s3cache.py

```diff
@@ -16,7 +16,7 @@
 import logging
 
 import boto3
-from werkzeug.contrib.cache import BaseCache
+from cachelib import BaseCache
```

Comment:
I'm pretty sure this should be `from cachelib.base import BaseCache`

Reply:
I _think_ the `BaseCache` module gets exposed at the cachelib level too if i'm reading this __init__ file correctly? https://github.com/pallets/cachelib/blob/master/cachelib/__init__.py

Reply:
I confirmed this via the console as well
![image](https://user-images.githubusercontent.com/7409244/82287117-47418400-9954-11ea-9a40-98be9cb60204.png)